src/nvim/ex_cmds2.c

```diff
@@ -1409,6 +1409,7 @@ void ex_listdo(exarg_T *eap)
     listcmd_busy = true;            // avoids setting pcmark below
 
     while (!got_int && buf != NULL) {
+      bool execute = true;
```

Comment:
The changes LGTM, but what's the reasoning behind calling this variable `execute`? Why not something like `focusable`? Also let's move the declaration of this variable to its first assignment at line 1438, which makes the code (slightly) easier to follow.

Reply:
> "Also let's move the declaration of this variable to its first assignment at line 1438, which makes the code (slightly) easier to follow."

The variable is used outside the scope of where it's first assigned, at line [1468](https://github.com/neovim/neovim/blob/f436506402f8e233f840f36f9a277c884cf6712b/src/nvim/ex_cmds2.c#L1468).

The variable was named `execute` for how it was used in the later part of the update (i.e., whether `do_cmdline` is called).

Reply:
> The variable is used at line 1468, outside the scope of where it's assigned at line 1438.

Oops, you're right :) Hard to visually inspect indent levels in the web interface sometimes...